enterprise_access/apps/subsidy_request/models.py

```diff
@@ -44,6 +46,12 @@ class SubsidyRequest(TimeStampedModel, SoftDeletableModel):
         max_length=128
     )
 
+    course_title = models.CharField(
+        null=True,
+        blank=True,
+        max_length=128
```

Comment:
[clarification] Is 128 characters the maximum a course title can be in course discovery? Or is this potentially too limiting?

Reply:
good call out. i'll check

Reply:
was 255: https://github.com/openedx/course-discovery/blob/ad1af6e122fc19250f85387d5d4447c48b42ff3e/course_discovery/apps/course_metadata/models.py#L778

change incoming...